The header defines USES_FHS, while some source files expect USE_FHS, so
define both to be safe.

--- tripwire-2.3.1-2/src/core/platform.h	Fri Aug 10 00:29:29 2001
+++ tripwire-2.3.1-2/src/core/platform.h	Fri Aug 10 00:29:36 2001
@@ -291,6 +291,7 @@
 //       where <tw-exe-dir> is the directory the tripwire executable is located in.
 //=============================================================================
 #define USES_FHS 				IS_LINUX
+#define USE_FHS 				IS_LINUX
 
 //=============================================================================
 // Miscellaneous
